tests/performance/apdex-score/README.md

```diff
@@ -10,141 +10,204 @@
+      "search": {
+        "value": "Alexandra Lemke",
+        "asserts": [
+          {
+            "selector": "//*[contains(@text, \"No reports found\")]"
+          }
+        ],
+        "postTestPath": [
+          { "selector": "//*[@text=\"Tasks\"]" }
+        ]
+      }
+    },
+    "taskList": {
+      "navigation": [
+        {
+          "selector": "//*[@text=\"Tasks\"]",
+          "asserts": [ { "selector": "//*[contains(@text, \"Danger sign follow up\")]" } ]
+        }
+      ]
+    },
+    "targets": {
+      "navigation": [
+        {
+          "selector": "//*[@text=\"Targets\"]",
+          "asserts": [ { "selector": "//*[contains(@text, \"New pregnancies\")]" } ]
+        }
+      ]
     }
   },
   
   "forms": {
     "patientReport": {
       "navigation": [{
-        "scrollDown": <Optional. Number of scrolls>,
-        "selector": "//*[@text=\"<a report form name>\"]"
+        "scrollDown": 3,
+        "selector": "//*[@text=\"Pregnancy\"]"
       }],
       "pages": [
         {
           "asserts": [
-            { "selector": "//*[contains(@text, \"<a text in the form>\")]" } // Wait for form to load
+            { "selector": "//*[contains(@text, \"Select patient\")]" } // Wait for form to load
           ],
           "fields": [
             {
-              "selector": <Mandatory. String, clickable element selector>, // Radio buttons, text boxes (to activate), etc.
-              // Use this when you need to type in the phone's keyboard, find the keycodes here: https://developer.android.com/reference/android/view/KeyEvent
-              "keycodes": <Optional. Array of numbers (keycodes).  
-              // Use this when you want an equivalent of element.setValue(myValue)
-              "value": <Optional. String or number>
-              // Use this when you want to select an option from a dropdown. The dropdown will open with the "selector" property
-              "dropdownOption": <Optional. String, clickable element selector>
-              // Scroll to reach field
-              "scrollDown": <Optional. Number of scrolls>
-              "scrollUp": <Optional. Number of scrolls>
-            },
+              "selector": "<Mandatory. XPath selector of radio buttons, text boxes (to open keyboard), etc.>",
+              // Optional. Use this when you need to type in the phone's keyboard, find the keycodes here: https://developer.android.com/reference/android/view/KeyEvent
+              "keycodes": "<Array of numbers (keycodes)>",  
+              // Optional. Use this when you want an equivalent of element.setValue(myValue)
+              "value": "<String. Field value>",
+              // Optional. Use this when you want to select an option from a dropdown. The dropdown will open with the "selector" property
+              "dropdownOption": "<String. XPath selector of dropdown>",
+              // Optional. Number of scrolls to reach the field
+              "scrollDown": 1,
+              "scrollUp": 1
```

Comment:
I am a little confused with this section.
For the `"selector":` do I need to add selectors for all the fields available in the form? and the same for the `"dropdownOption":`, should I add the value of any dropdown in the form? Or should I add selectors for all the dropdowns available in the form?

Reply:
This is inside `fields`, which is an array. Each element of the array is about 1 field. 
The "selector" is to click on that field and set focus: a radio, input text, dropdown, etc.
The "dropdownOption" is to click on the dropdown's option we want. It's better explained in the `Fields` section below L313. I missed updating it in this sample json, fixed!

For example, this is from MoH Kenya config form:
```
{
  "id": "county_residence",
  "selector": "//android.view.View[contains(@text, \"County of residence\")]",
   "dropdownOption": "//android.widget.CheckedTextView[@resource-id=\"android:id/text1\" and @text=\"BARINGO\"]"
},
```
It will click on the field `County of residence`, which will open the dropdown, then it will click on `BARINGO` option. 

The `selector` behaves the same across the whole settings file. It'll click on the selectors. This is convenient when you want to click on a label to remove the focus on a field, but this is usually not needed. I had to do it only once (Enketo being Enketo). 

There are some videos about[ this in here](https://drive.google.com/drive/folders/1d19YD3D2nxQkEorsd4QTnDK1UOPHS31U)

I think once we have more settings file examples, then we can remove this json from the REAME.

Reply:
😅  Thank you very much for the explanation Jenny!